### PR TITLE
Switch main's route handlers to be static functions

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: src/main/lens-api.ts [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import http from "http";
```
Comment (jim-docker) [inline src/main/lens-api.ts]: guess this file could be now named `src/main/utils/http-respond.ts` or similar?

**Review by jim-docker — DISMISSED** [review]
Why do we need this? Cleanup/modernize?

File: src/main/lens-api.ts (hunk 1 above)
Comment (Nokel81) [reply src/main/lens-api.ts]: Probably would be a better name.

**Comment by Nokel81** [comment]
> Why do we need this? Cleanup/modernize?

Mostly cleanup. These don't need to be instance methods. The fact that they are instance methods is slightly confusing because (one would hope) that all of them don't store state (even though I know that this change doesn't do anything to enforce this).

**Review by jim-docker — APPROVED** [review]

**Review by nevalla — APPROVED** [review]
LGTM

**Pull request merged**